### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by peterjuma**

### CODE REVIEW

**Review by jumapeter — APPROVED** [review]
Approved

**Review by jumapeter — APPROVED** [review]
Aprove update

**Review by pjuma — APPROVED** [review]
Approve

File: README.md [hunk 1]
```diff
@@ -4,3 +4,8 @@
 
 
 ### Yest another change
+
+## Changes...
+
```
Comment (peterjuma) [inline README.md:9]: Sed in libero ut nibh placerat accumsan.
Sed libero.
Nulla neque dolor, sagittis eget, iaculis quis, molestie non, velit.

Comment (peterjuma) [inline README.md:9]:
```suggestion
## Changes...

```

File: README.md [hunk 2]
```diff
@@ -4,3 +4,8 @@
 
 
 ### Yest another change
```
Comment (peterjuma) [inline README.md:6]: Some comment

Comment (peterjuma) [inline README.md:6]: Add

**Pull request closed without merging**